### PR TITLE
bringup tests on intel gpu

### PR DESCRIPTION
**Pull request by hfp**
Leverage Vulkan based runtime and bring-up initial CI. This work includes changes for tpp-scripts (not tracked here).

### CODE REVIEW

File: scripts/buildkite/tpp-gpu.yml [hunk 1]
```diff
@@ -20,7 +17,20 @@ steps:
...
+    env:
+      TPP_CLANGVER: "16.0.4"
+      TPP_GCCVER: "9.5.0" # Use an older version to match CUDA requirements
+      TPP_CUDAVER: "11.5"
+
+  - label: "TPP-MLIR-intel"
+    command: "${SRUN} -- \
+              'KIND=Debug COMPILER=clang LINKER=lld GPU=1 CHECK=1 \
+              $${BUILDKITE_BUILD_CHECKOUT_PATH}/scripts/buildkite/build_tpp.sh'"
+    env:
```
Comment (rengolin) [inline scripts/buildkite/tpp-gpu.yml:33]: I'd imagine the Intel version would need at least `TPP_CLANGVER`

Comment (hfp) [reply scripts/buildkite/tpp-gpu.yml:33]: The PR is not yet ready for review. It does not even build and is mostly about getting SSH-based execution on target system up.

Comment (rengolin) [reply scripts/buildkite/tpp-gpu.yml:33]: I get it, just making some en passant comments. :)

**Comment by hfp** [comment]
Unrelated note (TODO: Buildkite issue): apparently, "Prefix third-party fork branch names" in Buildkite makes agent unable to checkout the repository as 2nd time like "Retry" (and "Rebuild?).

**Comment by hfp** [comment]
Sending PR from forked repository does not work out for this subject.

**Pull request closed without merging**